model/recipe-model.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const recipeSchema = new Schema({
+  recipename: { type: String, required: true, unique: true},
```

Comment:
do we want this to be unique (e.g., no more than one chocolate cake recipe)?
also, we can add a few more items here (see sample JSON for example)